keel-artifact/src/main/kotlin/com/netflix/spinnaker/keel/artifact/ArtifactListener.kt

```diff
@@ -171,7 +168,7 @@ class ArtifactListener(
     return ArtifactStatus.valueOf(status)
   }
 
-  private fun Artifact.type() = ArtifactType.valueOf(type.toUpperCase())
+  private fun Artifact.type() = ArtifactType.valueOf(type.toLowerCase())
```

Comment:
It's slightly worrying how often we do this

Reply:
There's a kork `artifact` and the artifact we use. Both have a type. We only do this (I think) when we're working with or converting the kork artifact.

Reply:
I sure hope there aren't any times we do it that aren't covered by a test

Reply:
😬 me too.